eng/common/scripts/get-markdown-files-from-changed-files.ps1

```diff
@@ -0,0 +1,47 @@
+param (
+    # The root repo we scaned with.
+    [string[]] $RootRepo = "./",
+    # The target branch to compare with.
+    [string] $targetBranch = ${env:SYSTEM_PULLREQUEST_TARGETBRANCH}
+)
+$deletedFiles = (git diff origin/$targetBranch HEAD --name-only --diff-filter=D)
+$renamedFiles = (git diff origin/$targetBranch HEAD --diff-filter=R)
+$changedMarkdowns = (git diff origin/$targetBranch HEAD --name-only -- '*.md')
+
+$beforeRenameFiles = @()
+# Retrieve the renamed from files.
+foreach($file in $renamedFiles) {
+    if ($file -match "^rename from (.*)$") {
+        $beforeRenameFiles += $file -replace "^rename from (.*)$", '$1'
```

Comment:
While I'm this for now there are some changes that this can have reliability issues if the string output of git diff changes in some way.

Reply:
I agree. However, there is no git command which can return the file before rename. I will add comments here.